code/controllers/subsystems/falling.dm

```diff
@@ -0,0 +1,107 @@
+#define REMOVE_AND_CONTINUE			\
```

Comment:
Not vital, but the right side of this should probably be aligned with spaces instead of tabs so it looks OK on GitHub.